Documentation/ceph-dashboard-monitoring.md

```diff
@@ -0,0 +1,352 @@
+```yaml
+mgr:
+...
+  modules:
+    ...
+  - name: prometheus
+    enabled: true
+...
+dashboard:
+  enabled: true
+  ssl: true
+...
+monitoring:
+  enabled: true
+  rulesNamespace: rook-ceph  
+```
+
+## Prometheus Operator
+
+The Prometheus operator allows you to set up a Prometheus instance in you kubernetes cluster. Follow the instructions provided in [Prometheus Monitoring](https://github.com/rook/rook/blob/master/Documentation/ceph-monitoring.md).
+
+To verify that prometheus is running you can try to access the prometheus instance.
+
+Execute:
+```console
+echo "http://$(kubectl -n rook-ceph -o jsonpath={.status.hostIP} get pod prometheus-rook-prometheus-0):30900"
+```
+>```
+>http://192.168.122.70:30900
+>```
+If you use the output URL in your browser, you should see the Prometheus monitoring website.
+
+![Prometheus Monitoring Website](media/prometheus-monitor.png)
+
+## Prometheus Ceph manager module
+
+The source of the Ceph cluster metrics is this manager module, by default is enabled, and the Prometheus operator service monitor configures Prometheus to scrape data from it.
+
+In order to avoid monitoring issues just verify that the scrape intervals are the same in Prometheus and in the Ceph manager module. Both intervals must be the same. Depending on your monitoring needs, a value between 5 and 30 is correct.
+
+In the Prometheus web UI, go to Status-Configuration. The scrape configuration should be the following:
+
+```yaml
+scrape_configs:
+- job_name: rook-ceph/rook-ceph-mgr/0
+  honor_timestamps: true
+  scrape_interval: 5s
+```
+
+To verify this value in the Ceph Manager module, open a [Rook toolbox](ceph-toolbox.md) and execute:
+```console
+ceph config get mgr mgr/prometheus/scrape_interval
+```
+>```
+>30.000000
+>```
+
+If you have a different value (as you can see in this example), you can change it executing:
+```console
+ceph config set mgr mgr/prometheus/scrape_interval 5
+```
+
+## Grafana
+
+The Grafana image used is a _customized_ image of Grafana V.6.7.4, and it includes the set of Grafana dashboards used in the Ceph dashboard, and a couple of visualization plugins.
+
+In order to deploy Grafana properly configured for Ceph, we will use the grafana.yaml file stored in the monitoring folder. Some modifications in this file could be needed: 
+
+**1. Change of SSL certificates (optional)**
+
+The grafana.yaml file includes a configmap for the SSL certificates used to enable secure access to the UI. These certificates are self-signed and if it is needed or preferred can be replaced by others.
+
+**2. Grafana storage (mandatory)**
+
+Grafana needs storage for its database and other stuff. The "grafana.yaml" file provided uses a PVC that get the storage from a PV based in a Storage Class with name "ceph-storage". _The user must be sure that a PV using this storage class is available for the grafana PVC_. Or change the storage class used in the grafana PVC to point the right one for your cluster. 
```

Comment:
The PVs are expected to be dynamically provisioned with the csi driver. Why would there be a PV already available? The PVC will just request it to be created.